src/app/views/query-runner/query-runner.scss

```diff
@@ -7,3 +7,8 @@
   }
 }
 
+ #Dropdown17:focus {
```

Comment:
Does this id get generated each time as dropdown17? or would a new build make it dropdown20?

Reply:
Great question! 
Seems like it could actually change. Guess I'll have to find another way to manipulate it on focus 🤦🏾‍♀️

Reply:
Great question! 
Seems like it could actually change. Guess I'll have to find another way to manipulate it on focus 🤦🏾‍♀️